flake.nix

```diff
@@ -73,6 +73,23 @@
             );
           };
 
+        devShells.slim = with pkgs.ocamlPackages; pkgs.mkShell {
+          inputsFrom = [ dune_3 ];
+          nativeBuildInputs = with pkgs; [ pkg-config nodejs-slim ];
+          buildInputs = [
+            merlin
+            ocamlformat
+            ppx_expect
+            ctypes
+            integers
+            mdx
+            cinaps
+            menhir
+            odoc
+            lwt
```

Comment:
I guess this is only needed because dune-rpc-lwt isn't it in nixpkgs?

Reply:
I suppose that's the case. `make test` requests it:

```
File "otherlibs/dune-rpc-lwt/test/dune", line 17, characters 2-10:
17 |   lwt.unix
       ^^^^^^^^
Error: Library "lwt.unix" not found.
-> required by library "dune_rpc_lwt_tests" in
   _build/default/otherlibs/dune-rpc-lwt/test
-> required by
   _build/default/otherlibs/dune-rpc-lwt/test/.dune_rpc_lwt_tests.inline-tests/inline_test_runner_dune_rpc_lwt_tests.exe
-> required by alias otherlibs/dune-rpc-lwt/test/runtest in
   otherlibs/dune-rpc-lwt/test/dune:3
```

Reply:
Yeah, in the future it would be nice to get inputsFrom from all the packages in the repo.